frontend/admin/app/Http/Controllers/AuthController.php

```diff
@@ -20,6 +20,18 @@ public function login(Request $request)
             : null
         );
 
+        $ui_locales = null;
+        if(isset($_GET['locale']))
```

Comment:
With laravel, the usual way to get parameters is with `$request->input('locale')`. 
I think the main advantage is that it abstracts the source of the parameter - it works regardless of whether locale was a url parameter, a form input, or from a json body.

Reply:
Oh, that's much nicer, thanks.  It handle nulls without having to explicitly check, too.